src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -102,6 +102,7 @@ internal static class LogMessages
         public const string IDX10262 = "IDX10262: One of the issuers in TokenValidationParameters.ValidIssuers was null or an empty string. See https://aka.ms/wilson/tokenvalidation for details.";
         public const string IDX10263 = "IDX10263: Unable to re-validate with ConfigurationManager.LastKnownGoodConfiguration as it is expired.";
         public const string IDX10264 = "IDX10264: Reading issuer signing keys from validation parameters and configuration.";
+        public const string IDX10265 = "IDX10265: Network error occurred. Status code: '{0}'. \nResponse content: '{1}'. \nAttempting to retrieve configuration again from authority: '{2}'.";
 
```

Comment:
nit: If the response content can be long, consider moving it to the end.

Reply:
That's a good point; I imagine it's likely longer than the exception in string form. 

I think status code is something that should be present earlier on in the message though (for debugging purposes); do you think that separating 'status code' from 'response content' in the log message makes sense?

Reply:
I meant moving the authority to before the content, so it is easier to see if the content is long.
So the order will be {Status Code}, {Authority}, {Content}

Reply:
Makes sense. :) As of this commit https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1784/commits/daa8b451296c5da82fa6bcf59d176572ea67b6e0, the logging related to this has been moved to the HttpDocumentRetriever. The value of {content} is logged at the end.